security-framework/src/trust.rs

```diff
@@ -119,9 +119,9 @@ impl SecTrust {
     /// Returns a specific certificate from the certificate chain used to evaluate trust.
     ///
     /// Note: evaluate must first be called on the SecTrust.
-    pub fn certificate_at_index(&self, ix: i64) -> Option<SecCertificate> {
+    pub fn certificate_at_index(&self, ix: isize) -> Option<SecCertificate> {
         unsafe {
-            let certificate = SecTrustGetCertificateAtIndex(self.0, ix);
+            let certificate = SecTrustGetCertificateAtIndex(self.0, ix as CFIndex);
```

Comment:
No particular reason. Thought it was desirable to use 'rusty' types in the signature. I'll change it.